src/views/portals/HomeView.vue

```diff
@@ -0,0 +1,213 @@
+          <dashboard-images-gallery v-if="$screen.isMobile" />
+          <dashboard-routes-list />
+          <dashboard-articles-list v-if="$screen.isMobile" />
+          <useful-links v-if="$screen.isMobile" />
+          <forum-widget :message-count="20" v-if="$screen.isMobile" />
+        </div>
+      </div>
+      <div
+        v-if="!$screen.isMobile"
+        class="column"
+        :class="feed ? 'is-5-tablet is-5-desktop is-4-widescreen is-3-fullhd' : 'is-5 is-4-fullhd'"
+      >
+        <publi-widget />
+        <dashboard-images-gallery v-if="!feed" />
+        <useful-links />
+        <forum-widget :message-count="20" />
+        <dashboard-articles-list v-if="!feed" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import HomeBanner from './HomeBanner';
+import DashboardArticlesList from './utils/DashboardArticlesList';
+import DashboardImagesGallery from './utils/DashboardImagesGallery';
+import DashboardOutingsList from './utils/DashboardOutingsList';
+import DashboardRoutesList from './utils/DashboardRoutesList';
+import ForumWidget from './utils/ForumWidget';
+import PubliWidget from './utils/PubliWidget';
+import UsefulLinks from './utils/UsefulLinks';
+
+import FeedWidget from '@/components/feed-widget/FeedWidget';
+
+export default {
+  name: 'HomeView',
+
+  components: {
+    HomeBanner,
+    DashboardArticlesList,
+    DashboardImagesGallery,
+    DashboardOutingsList,
+    DashboardRoutesList,
+    ForumWidget,
+    PubliWidget,
+    UsefulLinks,
+    FeedWidget,
+  },
+
+  data() {
+    let state = true;
+
+    if (this.$user.isLogged) {
+      state = false;
+    }
+    return {
+      isPersonal: this.$localStorage.get('isPersonal', false),
+      feed: this.$localStorage.get('feed', false),
+      visible: state,
+    };
+  },
+
+  methods: {
+    toogleProperty(property) {
+      this.setProperty(property, !this[property]);
+    },
+    setProperty(property, value) {
+      this[property] = value;
+      this.$localStorage.set(`${property}`, this[property]);
+    },
+  },
+};
+</script>
+
+<style scoped lang="scss">
+@media screen and (max-width: $tablet) {
+  .feed-view {
+    padding-left: 0;
+    padding-right: 0;
+
+    .feed-title {
+      padding-left: 0;
+      padding-right: 0;
+    }
+  }
+  /*
+  .toggleContainer {
+    background: none !important;
+    border: none !important;
+  }
+
+  .is-active {
+    color: #f93 !important;
+    background: none !important;
+  }
+
+  .toggleContainer span {
+    padding: 3px 2px !important;
+  }*/
+}
+
+@media screen and (min-width: $tablet) {
+  .feed-view {
+    margin-top: 1.5rem;
+  }
+}
+
+.feed-title {
+  display: flex;
+
+  span:first-child {
+    flex-grow: 1;
+  }
+}
+
+h4 {
+  padding-bottom: 0.7rem !important;
+  margin-bottom: 0.7rem !important;
+  border-bottom: 1px solid #ddd;
+}
+
+.intro {
+  margin-bottom: 1rem;
+}
+
+.switch[type='checkbox']:checked + label::before {
+  background: #f93;
```

Comment:
isn't f93 a variable already?
please try to avoid magic numbers and values in CSS